### PR TITLE
[quantization] Move quantized::add_scalar to ATen

### PR DESCRIPTION
**Pull request by jamesr66a**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #27258 [quantization] Move quantized::mul_scalar to ATen
* **#27257 [quantization] Move quantized::add_scalar to ATen**

Differential Revision: [D17731566](https://our.internmc.facebook.com/intern/diff/D17731566)

### CODE REVIEW

File: aten/src/ATen/native/native_functions.yaml [hunk 1]
```diff
@@ -254,18 +255,36 @@
...
   use_c10_dispatcher: unboxed_only
   variants: method
   supports_named_tensor: True
+  dispatch:
+    CPU: add_
+    CUDA: add_
+
+- func: add_relu.Tensor(Tensor self, Tensor other) -> Tensor
+  use_c10_dispatcher: full
+  variants: function, method
```
Comment (dzhulgakov) [inline aten/src/ATen/native/native_functions.yaml]: don't make it a function and put into `nn` namespace

File: torch/nn/quantized/modules/functional_modules.py [hunk 2]
```diff
@@ -137,7 +137,7 @@ def add(self, x, y):
     r"""Operation equivalent to ``torch.ops.quantized.add(Tensor, float)``"""
     def add_scalar(self, x, y):
```
Comment (dzhulgakov) [inline torch/nn/quantized/modules/functional_modules.py]: I wonder whether we should drop it and just make a dispatch on "is scalar" inside the op. The difference is that the arguments would be different (passing or not qscale).

**Comment by raghuramank100** [comment]
Similar comments to add_mul PR, otherwise looks good

**Review by z-a-f — APPROVED** [review]
LGTM

**Pull request closed without merging**